script/DeployL1OptimismProposal.s.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/console.sol';
+import {Script} from 'forge-std/Script.sol';
+import {AaveGovernanceV2, IExecutorWithTimelock} from 'aave-address-book/AaveGovernanceV2.sol';
+
+library DeployL1OptimismProposal {
+  // address internal constant CROSSCHAIN_FORWARDER_OPTIMISM =
```

Comment:
seems not needed

Reply:
yes once deployed will inline the correct address here like on polygon